packages/i18nify-playground/src/constants/locale/index.js

```diff
@@ -0,0 +1,32 @@
+export const LANGS = [
+  {
+    value: 'en-IN',
+    label: 'Indian English',
+    icon: '/assets/icons/ic_flag_in.svg',
+  },
+  {
+    value: 'en-UK',
+    label: 'English',
+    icon: '/assets/icons/ic_flag_en.svg',
+  },
+  {
+    value: 'de-DE',
+    label: 'German',
+    icon: '/assets/icons/ic_flag_de.svg',
+  },
+  {
+    value: 'fr-FR',
+    label: 'France',
+    icon: '/assets/icons/ic_flag_fr.svg',
+  },
+  {
+    value: 'en-US',
+    label: 'USA',
+    icon: '/assets/icons/ic_flag_us.svg',
+  },
+  {
+    value: 'ms-MY',
+    label: 'MY',
+    icon: '/assets/icons/ic_flag_my.svg',
+  },
```

Comment:
Can we also include some other unique geographies like Baharain or so here or maybe include all supported geo's here? Thinking from an open-source pov.

Reply:
yep, we can plan this in next iteration. 
We can include a big list of locales in the dropdown